proto/celestia/blobstream/v1/tx.proto

```diff
@@ -1,13 +1,13 @@
 syntax = "proto3";
-package celestia.qgb.v1;
+package celestia.blobstream.v1;
```

Comment:
the path is included in the msg name, so changes this will result in different bytes included in any transactions or state. nodes with this change will know how to parse these txs, where nodes without the change will not

Reply:
therefore this can only be included in v2. It will also require a migration script, which I think we should include in this change. That likely means waiting for the upgrade and e2e code to be finished. 

Recall our conversaions about not being able to change the name of the qgb in the code before mainnet

Reply:
>  so changes this will result in different bytes included in any transactions or state. nodes with this change will know how to parse these txs, where nodes without the change will not

This provided people are sending register evm address transactions on mocha, which I assume is not the case. If not, then I can revert the tx related changes and only leave the query ones